src/main/java/business/NewsSources/NewsOutlet.java

```diff
@@ -0,0 +1,144 @@
+package business.NewsSources;
+
+import business.Sanitizer.*;
+import org.jsoup.nodes.*;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.time.LocalDateTime;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+
+import static business.Helper.Scraper.*;
+
+public abstract class NewsOutlet {
+    String name;
```

Comment:
should add access modifier: `private` or `protected`

Reply:
resolved